bluetti_mqtt/core/devices/eb3a.py

```diff
@@ -50,7 +50,7 @@ def polling_commands(self) -> List[QueryRangeCommand]:
         return [
             QueryRangeCommand(0x00, 0x0A, 0x28),
             QueryRangeCommand(0x00, 0x46, 0x15),
-            QueryRangeCommand(0x0B, 0xB9, 0x1C)
+            QueryRangeCommand(0x0B, 0xB9, 0x3D)
```

Comment:
So basically there's a trade-off between widening the range, which means more bluetooth packets have to be combined to reconstruct a single response and a potential risk of packet loss (which triggers a disconnect), or adding a second command, which adds a bit of latency. I don't have an EB3A to test with, which is why I think the current query looks like it does - either that or I had a temporary inability to do math 🤷‍♂️ . If you'd like to test it, I suspect the optimal solution might be something like:

```python
QueryRangeCommand(0x0B, 0xBF, 2),
QueryRangeCommand(0x0B, 0xDA, 1)
```